code/modules/clothing/spacesuits/rig/modules/utility.dm

```diff
@@ -117,8 +129,6 @@
 		device.afterattack(target,holder.wearer,1)
 	return 1
 
-
-
 /obj/item/rig_module/chem_dispenser
 	name = "mounted chemical dispenser"
```

Comment:
How is this different from the chem dispenser defined in `chem_dispenser.dm`?

Reply:
These are the de-selected files.